imagebuildah/stage_executor.go

```diff
@@ -348,6 +350,7 @@ func (s *StageExecutor) volumeCacheRestore() error {
 // imagebuilder tells us the instruction was "ADD" and not "COPY".
 func (s *StageExecutor) Copy(excludes []string, copies ...imagebuilder.Copy) error {
 	s.builder.ContentDigester.Restart()
```

Comment:
What's the impact of this getting called in between adding heredocs and adding other content on the history and build caching?

Reply:
Refactored now digester does not restarts between heredoc and regular copy. Before this it was affecting history of the image keeping only the most recent copy in the history.